nautobot/dcim/tables/template_code.py

```diff
@@ -92,7 +92,7 @@
 {% for i in record.tree_depth|as_range %}
     <i class="mdi mdi-circle-small"></i>
 {% endfor %}
-{{ record|hyperlinked_object }}
+<a href="{{ record.get_absolute_url }}">{{ record.name }}</a>
```

Comment:
Not blocking for this PR but we should think about enhancing `hyperlinked_object` to let you specify the display text, something like `{{ record|hyperlinked_object("name") }}` would be pretty convenient.